### DISCUSSION TITLE
HEX token has $0 Market Capital and 0 HEX In Circulation

### DISCUSSION THREAD

**Discussion by OneNewSmartIdea** [horizontalsystems/unstoppable-wallet-android#5859] [General]
Why HEX token has $0 Market Capital and 0 HEX In Circulation even thou most of the transactions are visible directly on the blockchain through DEX's and other tokens that are traded through CEX's have no problem showing these parameters?

**Comment by OneNewSmartIdea** [comment]
And the TVL is also 0

**Comment by esengulov** [comment]
@OneNewSmartIdea 

In both cases we merely aggregate these metrics (market cap and TVL)  from largest sources and display them in the app.

- price and market cap data obtained from coingecko API and it shows 0 amount for it. Likely a bug on their end.

- the TVL data obtained from Defillama API

**Reply by OneNewSmartIdea** [reply]
It's not a bug, they are willingly doing this to hide it from customers. CoinMarketCap stuck HEX at number 201 (on the 3rd page) since it was less than a penny instead of showing it in the first 10 coins, but they show the market cap and TVL. They do this because they don't own a stake in it and they promote their pumps and dumps. Yahoo Finance (which takes data from CoinMarketCap shows it in the right place on the first page, 8th place) and FORBES also shows it in the right place on the first page on the 7th place (they get their data from Nomics) 
Look for HEX here:
https://www.forbes.com/digital-assets/traded-assets/
https://finance.yahoo.com/crypto/
